packages/fast-components/src/text-area/fixtures/text-area.html

```diff
@@ -0,0 +1,83 @@
+<fast-design-system-provider style="font-family: var(--body-font)">
+    <h1>Text area</h1>
+    <h4>Default</h4>
+    <fast-text-area></fast-text-area>
+    <fast-text-area>
+        <span slot="label">label</span>
+    </fast-text-area>
+
+    <h4>Placeholder</h4>
+    <fast-text-area placeholder="Placeholder"></fast-text-area>
+
+    <!-- Required -->
+    <h4>Required</h4>
+    <fast-text-area required></fast-text-area>
+
+    <!-- Disabled -->
+    <h4>Disabled</h4>
+    <fast-text-area disabled></fast-text-area>
+    <fast-text-area disabled>
+        <span slot="label">label</span>
+    </fast-text-area>
+    <fast-text-area disabled placeholder="placeholder"></fast-text-area>
+
+    <!-- Read only -->
+    <h4>Read only</h4>
+    <fast-text-area readonly value="Readonly text area"></fast-text-area>
+    <fast-text-area readonly value="Readonly text area">label</fast-text-area>
+
+    <!-- Read only -->
+    <h4>Autofocus</h4>
+    <fast-text-area autofocus>autofocus</fast-text-area>
+
+    <!-- Resize -->
+    <h4>Resize</h4>
+    <h5>Both</h5>
+    <fast-text-area resize="both">resize both</fast-text-area>
+
+    <h5>Horizontal</h5>
+    <fast-text-area resize="horizontal">resize horizontal</fast-text-area>
+
+    <h5>Vertical</h5>
+    <fast-text-area resize="vertical">resize vertical</fast-text-area>
+
+    <h4>Filled</h4>
+    <h5>Default</h5>
+    <fast-text-area appearance="filled"></fast-text-area>
+    <fast-text-area appearance="filled">
+        <span slot="label">label</span>
+    </fast-text-area>
+
+    <h5>Placeholder</h5>
+    <fast-text-area appearance="filled" placeholder="Placeholder"></fast-text-area>
+
+    <!-- Required -->
+    <h5>Required</h5>
+    <fast-text-area appearance="filled" required></fast-text-area>
+
+    <!-- Disabled -->
+    <h5>Disabled</h5>
+    <fast-text-area appearance="filled" disabled></fast-text-area>
+    <fast-text-area appearance="filled" disabled>
+        <span slot="label">label</span>
+    </fast-text-area>
+    <fast-text-area appearance="filled" disabled placeholder="placeholder"></fast-text-area>
+
+    <!-- Read only -->
+    <h5>Read only</h5>
+    <fast-text-area appearance="filled" value="Readonly text area">label</fast-text-area>
```

Comment:
I seem to be able to edit the filled read only fields

Reply:
interesting, I'll check that out

Reply:
ah -specifically the filled ones

Reply:
oh, that's because it doesn't have the attr! Updating :)

Reply:
@eljefe223 resolved!